templates/news-detail/news-detail.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable function-paren-newline, object-curly-newline */
+import { div, h2, a, img } from '../../scripts/dom-helpers.js';
+import { getMetadata } from '../../scripts/aem.js';
+import { breadcrumbs } from '../../scripts/breadcrumbs.js';
+
+export default function decorate(doc) {
+  const $main = doc.querySelector('main');
+  const $picture = $main.querySelector('picture');
```

Comment:
we need to use `createOptimizedPicture` here.
Probably, we can also set `eager` flag to `true`. Not a concern right now but `LCP` is red for `hero image`

<img width="2672" alt="Screenshot 2024-12-09 at 12 54 11 PM" src="https://github.com/user-attachments/assets/795cb8a5-e88c-42f0-b0d1-15db52b37e2c">